doc/development_guide/testing.rst

```diff
@@ -76,7 +76,7 @@ The .rc file can also contain a section ``[testoptions]`` to pass options for th
 test runner. The following options are currently supported:
 
     "min_pyver": Minimal python version required to run the test
-    "max_pyver": Maximum python version required to run the test
+    "max_pyver": Python version from which the test won't be run. Setting to 3.10 will exclude 3.10.1 as well.
```

Comment:
```
Setting to 3.10 will exclude 3.10.1 as well.
```
I think that is obvious. I believe it would be more helpful to look at it from the other way round.

```
If the last supported version is 3.9 -> set it to 3.10
```

Reply:
```suggestion
    "max_pyver": Python version from which the test won't be run. If the last supported version is 3.9 this setting should be set to 3.10.
```